src/webviews/server.ts

```diff
@@ -89,6 +89,7 @@ export type OptionsToProtocol<O> = O extends WebviewCompileOptions<infer C, infe
     ? {
           submit: (result: S) => Promise<void> | void | never
           init: () => Promise<P | undefined> | P | undefined
+          getCompanyName: () => Promise<string>
```

Comment:
You'll still need to call this with `this.companyName = await client.getCompanyName()` or `client.getCompanyName().then(name => this.companyName = name)` in the `created` method of `samInvokeFrontend.ts`

We could make certain properties truly global within a webview. Just haven't gotten to that yet though.

Reply:
yeah this push was just to check CI :)